src/store/getters/state-getters.ts

```diff
@@ -533,8 +534,16 @@ export const isIncorporationApplicationValid = (state: StateIF): boolean => {
   )
 }
 
+/** Whether the partial registration steps are valid. */
+export const isPartialRegistrationValid = (state: StateIF): boolean => {
+  return (
+    getRegistration(state).defineBusinessValid &&
+    isAddPeopleAndRolesValid(state)
+  )
+}
+
```

Comment:
"isPartialRegistrationValid" only checks for the validities of the first two steppers.

Reply:
Curious about this, it appears we already have a getter for each respective step. What requirement is there that requests steps 1 and 2 validations in isolation?

Reply:
Sorry, reading further into the ticket/context, a firm registration is 2 steps?

Reply:
That is what I understand from this ticket. I'm not sure if this will be applied to all other filings.